test/test_helpers/publishing_api_test.rb

```diff
@@ -611,6 +611,30 @@
     end
   end
 
+  describe "#stub_publishing_api_graphql_query" do
```

Comment:
**thought:** testing test methods

Interesting that we're testing these test helper methods. I wonder how much this is just testing the library (webmock?)

Reply:
I did consider whether this test was needed, but concluded we're making a library of testing methods (with some logic, albeit very simple) available to other apps, so we should test them.